Source/JavaScriptCore/assembler/JITOperationList.h

```diff
@@ -36,7 +36,11 @@ namespace JSC {
 
 // This indirection is provided so that we can manually force on assertions for
 // testing even on release builds.
-#define JIT_OPERATION_VALIDATION_ASSERT_ENABLED ASSERT_ENABLED
+#if ENABLE(JIT_OPERATION_VALIDATION) && ASSERT_ENABLED
+#define JIT_OPERATION_VALIDATION_ASSERT_ENABLED 1
```

Comment:
Nit: but it might be nice to call this `ENABLE_JIT_OPERATION_VALIDATION_ASSERTS` so we can do: `ENABLE(JIT_OPERATION_VALIDATION_ASSERTS)` and you wouldn't need the `#define JIT_OPERATION_VALIDATION_ASSERT_ENABLED 0` below.

Reply:
I was thinking about that but wasn't sure if it was idiomatic to make an enable-flag outside of PlatformEnable.h and it didn't feel right to move this switch there
I'll do that and repost